<ui version="4.0" >
 <class>OsmaRenderDialog</class>
 <widget class="QDialog" name="OsmaRenderDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Osmarender rendering</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="MapStyle" >
     <property name="title" >
      <string>Output filename</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLineEdit" name="svgOutputFilename" />
        </item>
        <item>
         <widget class="QPushButton" name="btBrowseOutFilename" >
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Osmarender options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Zoom level</string>
              </property>
              <property name="buddy" >
               <cstring>sbZoom</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbZoom" >
              <property name="minimum" >
               <number>6</number>
              </property>
              <property name="maximum" >
               <number>17</number>
              </property>
              <property name="value" >
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Scale</string>
              </property>
              <property name="buddy" >
               <cstring>sbScale</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbScale" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>0.100000000000000</double>
              </property>
              <property name="maximum" >
               <double>5.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>min lat/Lon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbMinLat" >
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals" >
               <number>6</number>
              </property>
              <property name="minimum" >
               <double>-90.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>90.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbMinLon" >
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals" >
               <number>6</number>
              </property>
              <property name="minimum" >
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>180.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>max lat/Lon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbMaxLat" >
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals" >
               <number>6</number>
              </property>
              <property name="minimum" >
               <double>-90.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>90.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbMaxLon" >
              <property name="minimumSize" >
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals" >
               <number>6</number>
              </property>
              <property name="minimum" >
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>180.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="cbShowScale" >
            <property name="text" >
             <string>Show Scale</string>
            </property>
            <property name="tristate" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowGrid" >
            <property name="text" >
             <string>Show Grid</string>
            </property>
            <property name="tristate" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowBorders" >
            <property name="text" >
             <string>Show Borders</string>
            </property>
            <property name="tristate" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowLicense" >
            <property name="text" >
             <string>Show License</string>
            </property>
            <property name="tristate" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblInfoSVG" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreviewOK" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="cbShowPreview" >
          <property name="text" >
           <string>Generate PNG preview</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>DPI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbDPI" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="maximum" >
             <number>9999</number>
            </property>
            <property name="singleStep" >
             <number>5</number>
            </property>
            <property name="value" >
             <number>90</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbInfo" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreviewNOK" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>The Preview functionality requires Inkscape.
The path indicated in Tools-Preferences-[Tools] is not valid.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>379</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OsmaRenderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OsmaRenderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowPreview</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbDPI</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel" >
     <x>363</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
